Makefile

```diff
@@ -37,10 +37,12 @@ lint: lint-pylint lint-fmt
 lint-pylint:
 	pipenv run bandit -r $(dirs)
 	pipenv run pylint $(dirs)
+	pipenv run isort --profile=black --check-only $(dirs)
 
 lint-fmt:
 	@echo Checking python file formatting with the black code style checker
 	pipenv run black --line-length=100 --check $(dirs)
+	npx prettier . --check
```

Comment:
Does prettier have a `$(dirs)` flag similar to the other checks we can use?

Reply:
We could focus on those directories but since we're updating all of the Markdown/YAML files in the repo I think it makes sense to look at everything.

Reply:
Yeah, what Evan said 😄